ciao-image/datastore/store.go

```diff
@@ -178,11 +178,13 @@ func (s *ImageStore) UploadImage(ID string, body io.Reader) error {
 	if s.rawDs != nil {
 		err = s.rawDs.Write(ID, body)
 		if err != nil {
+			database.Logger.Errorf("Could not write image: %v", err)
```

Comment:
@obedmr yes you should do that. The reason why we did that was to avoid "import" issues with vendored version of glog in ciao and the upstream version of glog in another project using ciao.